### DISCUSSION TITLE
What does "Target peak height" mean?

### DISCUSSION THREAD

**Discussion by absay** [OpenTTD/OpenTTD#8898] [Q&A] [answered]
Sorry if this is not the right place to ask.

I'm translating the string `STR_MAPGEN_TERRAIN_TYPE_QUERY_CAPT` (`{BLACK}Target peak height`) but I'd like some clarification about what exactly this string is conveying since "target" can have several translations.

Thanks.

**Comment by nielsmh** [comment, marked as answer]
Intended goal for maximum height level that will be generated on the map, with the implication that this goal might not be hit exactly but the actual peak height of the map can be lower or higher than this.

**Comment by TrueBrain** [comment]
If someone has a better name suggestion, that is very welcome. We struggled with this for a bit of time :D

You can find this under `Terrain Type` -> `Custom` btw.
